waterdata/views.py

```diff
@@ -5,9 +5,40 @@
 from flask import render_template
 
 from . import app, __version__
+from .utils import get_water_services_data, parse_rdb
+
+
+water_services = app.config['WATER_SERVICES']
 
 
 @app.route('/')
 def home():
     """Render the home page."""
     return render_template('index.html', version=__version__)
+
+
+@app.route('/monitoring-location/<site_no>')
+def monitoring_location(site_no):
+    """
+    Monitoring Location view
+    :param site_no: USGS site number
+
+    """
+    rdb_resp_data = get_water_services_data(water_services, 'nwis/site/?site={}'.format(site_no))
+    content, status, reason = rdb_resp_data
+    if status == 200:
+        data = parse_rdb(content)[0]
+        station_name = data['station_nm']
+        return render_template('monitoring_location.html',
+                               status_code=status,
+                               station_name=station_name
+                               )
+    elif 400 <= status < 500:
+        return render_template('monitoring_location.html',
+                               status_code=status,
+                               reason=reason
+                               )
+    elif status == 500:
```

Comment:
Do we know what we can potentially get from the site service? This would imply that only a 500 would have us set a 502?

Reply:
That's probably a @JanellFry-usgs question.

@JanellFry-usgs: What are all the possible 5XX that the NWIS services can reply with?

Reply:
Refer to here to see all response codes the site service sends: https://internal.cida.usgs.gov/stash/projects/NWISWEB/repos/site-service/browse/src/main/java/gov/usgs/service/si/SIServlet.java#292-319

I like passing all the 400s on, but based on the logic discussed in the Slack channel I feel 503 would be best for everything else.

Reply:
@danielnaab -- Agreed.